packages/cardpay-reward-api/cardpay_reward_api/indexer.py

```diff
@@ -77,7 +77,7 @@ def add_root_and_proofs(self, db: Session, root, payment_list):
                 payee=payment["payee"],
                 proofArray=payment["proof"],
                 rewardProgramId=payment["rewardProgramID"],
-                amount=amount,
+                amount=str(amount),
```

Comment:
This makes our setup compatible with sqlite. Sqlite has a limitation of big numbers so must specify string. I don't have an issue with this. 

I think with this pr. The db can be replaced easily with an sqlite file

Reply:
@IanCal I know u were investigating this earlier. Is there anything changes u made to make the orm work for postgres and sqlite that I shud know about?

Reply:
@tintinthong There was something around dates, I think a datetime rather than time was one but you've already addressed that. String conversion was one that had to happen explicitly (as here) - wasn't much that needed changing.